app/controllers/submissions_controller.rb

```diff
@@ -199,16 +202,30 @@ def download
 
           position = annotation.coordinate.split(',')
           page  = position[2].to_i
+
+          width = position[3].to_f * pdf.bounds.width
```

Comment:
Would you mind assigning these array values to variables, or at least write a comment mapping out what values they contain.